components/cdc/src/service.rs

```diff
@@ -195,132 +266,240 @@ impl Service {
             memory_quota,
         }
     }
-}
 
-impl ChangeData for Service {
-    fn event_feed(
+    // Parse HTTP/2 headers. Only for `Self::event_feed_v2`.
+    fn parse_headers(ctx: &RpcContext<'_>) -> Result<EventFeedHeaders, String> {
+        let mut header = EventFeedHeaders {
+            version: semver::Version::new(0, 0, 0),
+            features: vec![],
+        };
+
+        let metadata = ctx.request_headers();
+        for i in 0..metadata.len() {
+            let (key, value) = metadata.get(i).unwrap();
+            if key == EventFeedHeaders::VERSION_KEY {
```

Comment:
There is already a version field in `ChangeDataRequest`
https://github.com/pingcap/kvproto/blob/961f64db1b7be036ebd3f1276fef5944a812755f/proto/cdcpb.proto#L21